.github/workflows/auto-update-i18n.yml

```diff
@@ -45,6 +47,7 @@ jobs:
         uses: EndBug/add-and-commit@v9
         with:
           add: '.'
+          default_author: github_actions
           message: '🤖 Updated i18n files'
           push: true
```

Comment:
and i think neither `push` nor `pull` are needed, i don't see in the other usages

Reply:
ic-js/.github/workflows/docs.yml uses a faulty "pull" because they don't want to pull. "Default" value is not the default so I think we can leave it like that. Push could be left out, but I feel like for someone who reads the code for a first time it makes sense to see what this action does. Idk no strong opinion on that one.

Reply:
ah understood! then, maybe, could you please adjust the snapshot workflow too in another PR?

Reply:
just add the push / pull to the snapshot workflow?